src/runtime/schema.ts

```diff
@@ -8,21 +8,25 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
-import {EntityClass, Entity} from './entity.js';
 import {ParticleExecutionContext} from './particle-execution-context.js';
-import {EntityType, Type} from './type.js';
 import {Dictionary} from './hot.js';
 import {CRDTEntity, SingletonEntityModel, CollectionEntityModel} from './crdt/crdt-entity.js';
 import {Referenceable} from './crdt/crdt-collection.js';
 import {CRDTSingleton} from './crdt/crdt-singleton.js';
 import {Flags} from './flags.js';
 
+// tslint:disable-next-line: no-any
+type SchemaMethod  = (data?: { fields: {}; names: any[]; description: {}; }) => Schema;
```

Comment:
Are names not always strings?

Reply:
Dunno. I just copied the signature that was already there. I'd rather not change that now.

Reply:
Sure, we can revisit our 'any's later.